import_logs.py

```diff
@@ -2098,6 +2123,31 @@ def detect_format(file):
         logging.debug('Format %s is the best match', format.name)
         return format
 
+    def is_filtered(self, hit):
+        host = None
+        if hasattr(hit, 'host'):
+            host = hit.host
+        else:
+            try:
+                host = urlparse.urlparse(hit.path).hostname
+            except:
+                pass
+
+        if host:
+            if config.options.exclude_host and len(config.options.exclude_host) > 0 and host in config.options.exclude_host:
+                return True
+
+            if config.options.include_host and len(config.options.include_host) > 0 and host not in config.options.include_host:
+                return True
+
+        if config.options.exclude_older_than and hit.date < config.options.exclude_older_than:
```

Comment:
Is it possible that hit.date is not set as date assignment is wrapped in a try/catch? Not sure what should happen if hit.date is not set... I think as long as this comparison does not result in an error it should be fine... 

actually, I see further below it calls `hit.date.date()` so we can likely assume the date is already always set.

However, I think you want to use `>` instead of `<` and also swap it for newer comparison. I did eg `./import_logs.py --dry-run --url=http://apache.piwik --exclude-older-than="2017-11-10 00:00:23" ../../test4939.log` which should have removed no entry as all log files are from Nov 19th. However, it imported all. Then I did `./import_logs.py --dry-run --url=http://apache.piwik `--exclude-newer-than="2017-11-10 00:00:23" ../../test4939.log` and it imported none. This was when using `dry-run` feature and I can send you the log file if needed. The comparison looks correct so wondering if it might be different issue.

Reply:
FYI: I debugged it and I think it works. The problem may be that `hit.date` as defined in log eg `[19/Nov/2017:00:00:01 +0100]` will be converted to UTC or something, while the excluded dates are not.

I reckon this is kind of expected but maybe also a little confusing? As a user someone will need to be careful. Maybe we could require to also specify timezone with the excluded dates or so? Any thoughts @diosmosis @mattab

Reply:
Simplest solution is to say the times must be in UTC in the help output.

Reply:
Suggestions:

* The excluded date should support the notation eg. `19/Nov/2017:00:00:01 +0100` or any other valid datetime such as `19/Nov/2017:00:00:01 NZST`  and convert it to UTC
* To clarify and avoid any user error I suggest we require user to input the timezone in the parameter. if timezone is missing, throw an error.

Reply:
FYI parsing timezone info is not available in python's std lib: https://stackoverflow.com/questions/13182075/how-to-convert-a-timezone-aware-string-to-datetime-in-python-without-dateutil

Can assume an external library like `dateutil` is available (requiring users to have it installed), use an email date format like `Tue May 08 15:14:45 +0800 2012`, force the timezone value to be in a numeric format (so `NZST` will not be supported), or find a timezone parser and include it with this repo. What do you want me to do?

Reply:
We don't need timezone but similar to log format be able to specify `+/-XXXX` as in `19/Nov/2017:00:00:01 +0100` to match same date as specified in the logs. This way a user basically doesn't need to think and transform the `+0100` manually.

Reply:
Then I'll require that a numeric timezone format is used.

Reply:
Awesome 👍